src/Hungary/Mews.Fiscalizations.Hungary.Tests/InvoiceTests.cs

```diff
@@ -266,7 +266,11 @@ private async Task AssertInvoices(ResponseResult<string, ResultErrorCode> sendIn
             }
 
             var validationResults = invoiceStatuses.SelectMany(s => s.Value.ValidationResults);
-            Assert.IsEmpty(validationResults);
+            Assert.IsEmpty(validationResults, "Response contains validation errors.", new
+            {
+                Message = String.Join(", ", validationResults.Select(r => r.Message)),
+                Code = String.Join(", ", validationResults.Select(r => r.ResultCode.ToString()))
+            });
```

Comment:
unrelated change but the tests keep failing, apparently they introduced some new validation :D 

anyway, in Mews we only support HUF for this integration so i believe we should be good.